app/code/core/Mage/Adminhtml/Block/Permissions/Tab/Rolesedit.php

```diff
@@ -108,20 +108,19 @@ public function __construct()
         $undefinedResources = array_diff(array_keys($resources), array_keys($resourcesPermissionsMap));
 
         foreach ($undefinedResources as $undefinedResourceId) {
-            if ($this->_getFallbackHelper()->fallbackResourcePermissions(
+            # Fallback resource permissions
```

Comment:
`#` for comments?

Reply:
I use it for singe linge comments ... one char less ;)

(Can you please suggest a change?)

Reply:
I prefer `//` but this is not important.

Reply:
Will change in next PR.